index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div class="container columns">
+      <nav id="navbar" class="column is-2">
+        <header>Navigation</header>
+        <a href="#JavaScript_and_Java" class="nav-link">JavaScript and Java</a>
+        <a href="#JavaScript_and_Java1" class="nav-link"
+          >JavaScript and Java1</a
+        >
+        <a href="#JavaScript_and_Java2" class="nav-link"
+          >JavaScript and Java2</a
+        >
+        <a href="#JavaScript_and_Java3" class="nav-link"
+          >JavaScript and Java3</a
+        >
+        <a href="#JavaScript_and_Java4" class="nav-link"
+          >JavaScript and Java4</a
+        >
+      </nav>
+      <main id="main-doc">
+        <section
+          class="main-section section is-medium"
+          id="JavaScript_and_Java"
+        >
+          <header id="JavaScript_and_Java">JavaScript and Java</header>
```

Comment:
<section> and <header> elements have the same id attribute value. Please give them different names. Ids should be unique. Can read more here ---> [id](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id)

Reply:
- [ ] Try to overhaul your review from grammar errors before submitting it, and remember that the review comment is giving changes in a polite way ^_^